### PR TITLE
Roster management

### PR DESCRIPTION
**Pull request by kcranston**
Adds a new script, `abc-roster` that creates a nbgrader-formatted roster from a GitHub Classsroom roster. Creates an `id` column from the `github_username` column, and creates `first_name`, `last_name` from `name`. Saves new file as `course_materials/nbgrader_roster.csv`. This addresses #59 (finally!).

Run `abc-roster -h` for usage. 

Also changes code and files to use `classroom_roster.csv` consistently in example files and config, #383 

Also adds better error handing for missing rosters, #384 (noting that I could not reproduce the error in this issue, so I am not 100% sure it it solved).

### CODE REVIEW

**Comment by nkorinek** [comment]
@kcranston Hey! Tested this one out a bit as well, it looks great! One thing I've noticed. It looks like in your code you have an exception that's supposed to trip if a bad column name is given (i.e., a name column that doesn't exist in the source file). When I passed a bad column name it would just run through and not pull up an error. This just left the first/last name columns empty. The custom column works if you give it a correct column name, which is good! Just doesn't throw the error when it should. Aside from that I test renaming it and tried breaking it but everything else seemed to work as expected and/or threw the errors I would expect. Hope this helps!

**Comment by kcranston** [comment]
Thanks, @nkorinek ! That exception trips when the `github_username` column does not exist. 

I've added a check for the `column_to_split` and if that column does not exist:
1. the script prints a warning message
2. it does not create blank `first_name` and `last_name` columns

**Comment by nkorinek** [comment]
Just checked and it seems like this works as you expected, good too go hopefully!

File: docs/get-started/configuration.rst [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
+roster
+======
+
+Enter the path to the classroom roster. If you enter a bare filename,
+`abc-classroom` will look in the course directory for the file. If your
+roster file is somewhere else, you can enter a full path to the file,
+e.g. `/home/user/directory/filename.csv`. 
+
+Default: `roster: classroom_roster.csv`
+=======
```
Comment (nkorinek) [inline docs/get-started/configuration.rst:13]: This is causing the build to fail, I don't think this underline should be here? If it should be it needs to be longer to underline the whole line.

**Comment by kcranston** [comment]
Fixed the syntax error that was causing the test failures. This is ready to merge now!

**Pull request merged**